### PR TITLE
Resolve table name with schema

### PR DESCRIPTION
**Pull request by mieciu**

### CODE REVIEW

File: quesma/clickhouse/quesma_communicator.go [hunk 1]
```diff
@@ -160,7 +160,6 @@ func executeQuery(ctx context.Context, lm *LogManager, query *model.Query, field
 	//
 
 	settings := make(clickhouse.Settings)
-	settings["readonly"] = "1"
```
Comment (nablaone) [inline quesma/clickhouse/quesma_communicator.go:163]: Why is that?

**Review by nablaone — APPROVED** [review]
LGTM.

File: quesma/clickhouse/quesma_communicator.go (hunk 1 above)
Comment (mieciu) [reply quesma/clickhouse/quesma_communicator.go:163]: Apparently that causes troubles with Hydrolix queries - I'll link you some stack traces.

Comment (pdelewski) [inline quesma/clickhouse/quesma_communicator.go:163]: Please add comment

**Review by pdelewski — APPROVED** [review]

**Pull request merged**